### ISSUE TITLE
Bugs in games

### ISSUE THREAD

**Issue by drhelius** [drhelius/Geargrafx#7]
- Order of the Griffon: Changes resolution in mid frame for character portraits.

**Comment by jbrandwood** [comment]
You may find some of the these test examples useful for figuring out some of the VDC's timing intricacies that can cause glithches in games ... https://github.com/pce-devel/huc/tree/master/examples/asm/elmer

**Comment by drhelius** [comment]
> You may find some of the these test examples useful for figuring out some of the VDC's timing intricacies that can cause glithches in games

It definitely helps. Thank you!

**Comment by Ryusennin** [comment]
Hi! Another issue is that 336x224 and 512x224 resolutions seem to be forced in 256x224. About 10% of PCE games use 336x224.

**Comment by drhelius** [comment]
> Hi! Another issue is that 336x224 and 512x224 resolutions seem to be forced in 256x224. About 10% of PCE games use 336x224.

I'm not sure I understand you.

You may refer to the aspect ratio forcing it to 4:3 by default? Try using square pixels.

**Comment by Ryusennin** [comment]
> You may refer to the aspect ratio forcing it to 4:3 by default? Try using square pixels.

I was actually using square pixels, but the emulator window doesn't automatically resize when games use different resolutions — which made me think the screen was truncated to 256x224. I switched to 4:3 ratio and it fixes the issue for 256x224 and 336x224 resolutions.

By the way, TV Sports Basketball (the only HuCard game I know of with some screens in 512x224) doesn't seem to work in GearGrafx. It boots on a black screen without any sound.

**Comment by drhelius** [comment]
> I was actually using square pixels, but the emulator window doesn't automatically resize when games use different resolutions — which made me think the screen was truncated to 256x224. I switched to 4:3 ratio and it fixes the issue for 256x224 and 336x224 resolutions.

You are right, the window does never resize. Based on the window size the content is resized to accommodate to the game resolution and your settings like integer scale, aspect ratio and so on. If you have the `Integer Scale (Manual)` setting set the content may be cropped by the window size. There is an option in the video menu `Resize Window to Content` to force a resize of the window to fit the content perfectly.

I recommend using `Integer Scale (Auto)` or `Scale to Window Height` so the content always respects the window size, and then adjust the window to your likings.

**Comment by drhelius** [comment]
I think that Gearcoleco can be marked as compatible now. Feel free to tag me on Discord if there is any issue or complain, I'm in the RetroAchivements.org and libretro servers.

Regarding SG-1000, there is support in Gearsystem for most games and I see what is missing for others.  I will work on it a let you know when it is finished.
